web/src/views/events/RecordingView.tsx

```diff
@@ -314,7 +314,10 @@ export function RecordingView({
       return undefined;
     }
 
-    const aspect = camera.detect.width / camera.detect.height;
+    const aspect =
```

Comment:
I think we can use `getCameraAspect` here instead of redoing the logic?

Reply:
for sure, good catch